### PR TITLE
Example improvement in Databox and EdgeOrder

### PR DESCRIPTION
**Pull request by dhawal777**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by isra-fel — DISMISSED** [review]
LGTM

File: src/DataBox/docs/New-AzDataBoxJob.md [hunk 1]
```diff
@@ -68,12 +69,24 @@ AddressType City          CompanyName Country PostalCode StateOrProvince StreetA
...
+  "Error": {
+    "Code": "StaticValidationGenericCountryCodeHasInvalidLength",
+    "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.",
+    "Details": [
+      null
+    ],
+    "Target": null
+  }
+} 
+```
```
Comment (isra-fel) [inline src/DataBox/docs/New-AzDataBoxJob.md:84]: The tool to process help document does not like multiple code blocks in one example unfortunalty. @dhawal777  Would you mind revising it as below?
```suggestion
You can expand and visualize other object in similar way how details and shipping address expanded, Also you can use `$DebugPreference = "Continue"`, with any example/usecase to get exact details of error in below format when command fails.

> {
>   "Error": {
>     "Code": "StaticValidationGenericCountryCodeHasInvalidLength",
>     "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.",
>     "Details": [
>       null
>     ],
>     "Target": null
>   }
> } 
```

Comment (dhawal777) [reply src/DataBox/docs/New-AzDataBoxJob.md:84]: Will appear like this , can we have some way to keep format intact

{ "Error": { "Code": "StaticValidationGenericCountryCodeHasInvalidLength", "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.", "Details": [ null ], "Target": null } }

Comment (isra-fel) [reply src/DataBox/docs/New-AzDataBoxJob.md]: Unless we add it to the example code as comments, like this:

### Example 1: Create a databox import job 
```powershell
PS C:\> $DebugPreference = "Continue"
***********
# You can expand and visualize other object in similar way how details and shipping address expanded, Also you can use `$DebugPreference = "Continue"`, with any example/usecase to get exact details of error in below format when command fails.
# {
#   "Error": {
#     "Code": "StaticValidationGenericCountryCodeHasInvalidLength",
#     "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.",
#     "Details": [
#       null
#     ],
#     "Target": null
#   }
# } 
```

What do you think?

Comment (dhawal777) [reply src/DataBox/docs/New-AzDataBoxJob.md]: Yes LGTM, Thanks

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**